core/src/test/java/com/sequenceiq/cloudbreak/converter/stack/cluster/ClusterToClusterV4ResponseConverterTest.java

```diff
@@ -44,13 +54,17 @@
 import com.sequenceiq.cloudbreak.service.secret.model.SecretResponse;
 import com.sequenceiq.cloudbreak.util.StackUtil;
 import com.sequenceiq.cloudbreak.workspace.model.Workspace;
+import com.sequenceiq.common.model.FileSystemType;
 
 @RunWith(MockitoJUnitRunner.class)
 public class ClusterToClusterV4ResponseConverterTest extends AbstractEntityConverterTest<Cluster> {
 
     @InjectMocks
     private ClusterToClusterV4ResponseConverter underTest;
 
+    @InjectMocks
+    private CloudStorageConverter cloudStorageConverter = new CloudStorageConverter();
```

Comment:
If I understand correctly, do you want to use the real functionality for this class in the tests?

Reply:
The test of real implementation of CloudStorageConverter is in CloudStorageConverterTest. I don't need to use the real functionality of CloudStorageConverter here. So I have updated the test to mock the implementation of CloudStorageConverter.